doc/man3/EVP_MD_fetch.pod

```diff
@@ -11,60 +11,88 @@ EVP_MD_fetch
 
  EVP_MD *EVP_MD_fetch(OPENSSL_CTX *ctx, const char *algorithm,
                       const char *properties);
+ EVP_CIPHER *EVP_CIPHER_fetch(OPENSSL_CTX *ctx, const char *algorithm,
+                              const char *properties);
 
 =head1 DESCRIPTION
 
-The B<EVP_MD> object is used for representing a digest method implementation.
+Cryptographic algorithms are represented by different OpenSSL objects depending
+on what type of algorithm it is.
 
-Having obtained a digest implementation as an B<EVP_MD> type it can be used to
-calculate the digest of input data using functions such as
-L<EVP_DigestInit_ex(3)>, L<EVP_DigestUpdate(3)> and L<EVP_DigestFinal_ex(3)>.
+The B<EVP_MD> object is used for representing a digest algorithm.
 
-Digest implementations may be obtained in one of three ways, i.e. implicit
-fetch, explicit fetch or user defined.
+B<EVP_CIPHER> objects are used to represent symmetric cipher algorithms.
+
+The algorithm objects may or may not have an associated algorithm
+implementation.
+Algorithms are implemented by providers and any one algorithm could have an
```

Comment:
What about something simpler like-
'Cryptographic algorithms are implemented by providers. Any algorithm may be supported by zero or more providers.'

Reply:
Done